### PR TITLE
[HYDRATOR-50] Hydrator UI: Sort order in Pipelines view is not saved

### PR DESCRIPTION
**Pull request by sushil-car**
https://issues.cask.co/browse/HYDRATOR-50

### CODE REVIEW

**Comment by sushil-car** [comment]
Changed the cask-sortable directive so that it will maintain column's sort order after page refresh
Also provided option to skip a column from getting sort, for this added the attribute skip-sort to be provided as skip-sort="skip" on column **th** that we don't want to sort. The sort icon for this **th** is made hidden by applying css 

This pull request is dependant on PR:
https://github.com/caskdata/cdap/pull/7529

File: modules/cask-angular-sortable/sortable.js [hunk 1]
```diff
@@ -47,17 +47,29 @@ function caskSortableDirective ($log) {
...
           defaultReverse = (a==='reverse');
         }
       });
 
-      if(!defaultPredicate) {
+      if (!defaultPredicate) {
         defaultPredicate = headers.eq(0);
       }
 
+      $stateParams.sortBy = defaultPredicate.attr('data-predicate');
```
Comment (ajainarayanan) [inline modules/cask-angular-sortable/sortable.js]: We don't need to do this I believe. ```$state.go(...)``` will update the `$stateParams` right? We should instead update this is in the object we pass in go.

Something like 

```
$state.go($state.$current.name, { 
  sortBy:  defaultPredicate.attr('data-predicate'),
  reverse: defaultReverse ? 'reverse' : '"
}, {
  notify: false
})
```

File: modules/cask-angular-sortable/sortable.js [hunk 2]
```diff
@@ -89,6 +105,10 @@ function caskSortableDirective ($log) {
             th.addClass('predicate');
           }
         });
+
+        $stateParams.sortBy = predicate;
```
Comment (ajainarayanan) [inline modules/cask-angular-sortable/sortable.js]: Same here.

**Comment by ajainarayanan** [comment]
Also this should have the zip file updated. You need to do 
```
gulp zip --module=cask-angular-sortable
```

**Comment by russorat** [comment]
@sushiladokar please address the comments here

**Comment by ajainarayanan** [comment]
👍 LGTM Please squash the commits before merge.

**Comment by sushil-car** [comment]
Squashing the commits

**Pull request merged**